components/Activity/AgendaList.tsx

```diff
@@ -0,0 +1,82 @@
+import { Agenda } from '@open-source-bazaar/activityhub-service';
+import { computed } from 'mobx';
+import { observer } from 'mobx-react';
+import { ObservedComponent } from 'mobx-react-helper';
+import { Column, RestTable } from 'mobx-restful-table';
+import { Badge } from 'react-bootstrap';
+
+import { AgendaModel } from '../../models/Agenda';
+import { i18n, I18nContext } from '../../models/Translation';
+import userStore from '../../models/User';
+import { renderTagInput } from '../Tag';
+
+export interface AgendaListProps {
+  activityId: number;
+}
+
+@observer
+export class AgendaList extends ObservedComponent<AgendaListProps, typeof i18n> {
+  static contextType = I18nContext;
+
+  agendaStore = new AgendaModel(this.props.activityId);
+
+  @computed
+  get columns(): Column<Agenda>[] {
+    const { t } = this.observedContext;
+
+    return [
+      {
+        renderHead: t('title'),
+        renderBody: ({ forum }) => forum?.title || t('unknown'),
+      },
+      {
+        renderHead: t('summary'),
+        renderBody: ({ forum }) => forum?.summary || '-',
+      },
+      {
+        renderHead: t('start_time'),
+        renderBody: ({ forum }) => forum?.startTime ? new Date(forum.startTime).toLocaleString() : '-',
+      },
+      {
+        renderHead: t('end_time'),
+        renderBody: ({ forum }) => forum?.endTime ? new Date(forum.endTime).toLocaleString() : '-',
+      },
+      {
+        renderHead: t('place'),
+        renderBody: ({ forum }) => forum?.place?.name || t('unknown'),
+      },
+      {
+        key: 'mentors',
+        renderHead: t('mentors'),
+        renderBody: ({ mentors }) => 
+          mentors?.map(mentor => mentor.name).join(', ') || '-',
+        renderInput: renderTagInput(userStore),
```

Comment:
```suggestion
        key: 'start_time',
        renderHead: t('start_time'),
        renderBody: ({ forum }) => forum?.startTime ? new Date(forum.startTime).toLocaleString() : '-',
        type: 'datetime-local',
      },
      {
        key: 'end_time',
        renderHead: t('end_time'),
        renderBody: ({ forum }) => forum?.endTime ? new Date(forum.endTime).toLocaleString() : '-',
        type: 'datetime-local',
      },
      {
        key: 'forum',
        renderHead: t('forum'),
        renderBody: ({ forum }) => forum?.name || t('unknown'),
        renderInput: renderTagInput(forumStore, { activity: activityId }),
        required: true,
        invalidMessage: t('field_required'),
      },
      {
        key: 'mentors',
        renderHead: t('mentors'),
        renderBody: ({ mentors }) => 
          mentors?.map(mentor => mentor.name).join(', ') || '-',
        renderInput: renderTagInput(userStore),
        required: true,
        invalidMessage: t('field_required'),
```
如果不加表单配置属性，集成表格组件就无法完全替代独立的编辑页。请自查所有 `editable` 表格相关配置，补充每个字段的表单属性。